kuma/wiki/tests/test_signals.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from kuma.wiki.models import Revision, Document
+
+
+@pytest.mark.tags
+def test_document_tags_cache_invalidated_correctly(root_doc, wiki_user):
```

Comment:
I think this could live in ``kuma/wiki/tests/test_models_document.py``, but it is OK here as well. However, you need to change the test name to say you are testing the ``on_document_save`` signal handler if you want to keep it here.